source/index.ts

```diff
@@ -1,7 +1,7 @@
-// NOTE -- 	
+// NOTE --
```

Comment:
I don't know how my editor let me commit with this... I have it setup to strip trailing whitespace on file save.  Either way, I support this change.  😁